plugins/nodes/src/js/pages/nodes/nodes-table/NodesTableContainer.js

```diff
@@ -0,0 +1,76 @@
+import mixin from 'reactjs-mixin';
+import React from 'react';
+import {StoreMixin} from 'mesosphere-shared-reactjs';
+
+import CompositeState from '../../../../../../../src/js/structs/CompositeState';
+import NodesTable from '../../../components/NodesTable';
+import QueryParamsMixin from '../../../../../../../src/js/mixins/QueryParamsMixin';
+
+class NodesTableContainer extends mixin(StoreMixin, QueryParamsMixin) {
+
+  constructor() {
+    super(...arguments);
+
+    this.state = {
+      filters: {health: 'all', name: '', service: null},
+      filteredNodes: [],
+      nodeHealthResponseReceived: false
```

Comment:
Just pointing out to rename this one too if you agree with my other comment. `receivedNodeHealthResponse`